tests/test_extension.py

```diff
@@ -0,0 +1,35 @@
+from unittest import mock
+import pytest
+
+from scrapy.crawler import Crawler
+from scrapy import Spider
+from spidermon.contrib.scrapy.extensions import Spidermon
+
+
+@pytest.fixture
+def spidermon_enabled_settings():
+    return {
+        'SPIDERMON_ENABLED': True,
+        'EXTENSIONS': {
+            'spidermon.contrib.scrapy.extensions.Spidermon': 500,
+        }
+    }
+
+
+@pytest.fixture
+def crawler(spidermon_enabled_settings):
+    crawler = Crawler(Spider, settings=spidermon_enabled_settings)
+    crawler.spider = Spider('dummy')
+    return crawler
+
+
+def test_engine_stopped_suites_should_run(crawler):
+    "The suites defined at engine_stopped_suites should be loaded and run """
```

Comment:
Maybe I'm missing something, but there shouldn't be `""" some text """`

Reply:
I believe that when black is executed, this will be fixed.